lib/tools/broccoli/node-modules-dest-copy.ts

```diff
@@ -69,7 +70,7 @@ export class DestCopy implements IBroccoliPlugin {
 				});
 			}
 		});
-		if (!_.isEmpty(this.dependencies)) {
+		if (!_.isEmpty(this.dependencies) && !this.$config.debugLivesync) {
```

Comment:
I believe we can find a better way of controlling the behavior than making specific exceptions for every case that comes up. This will be a heavy burden on us in the future, makes the code hard to read and understand and put's a strain on future development.